app/controllers/application_controller.rb

```diff
@@ -98,4 +98,24 @@ def track_visit(*args)
   def suppress_hotline_link
     @suppress_contact_us_message = true
   end
+
+  def disable_analytics
+    @disable_analytics = true
+  end
+
+  def disable_phase_banner
+    @disable_phase_banner = true
+  end
+
+  def disable_navigation
```

Comment:
I cannot see where `disable_navigation` is being used. Is it necessary?

Reply:
I could be reading this wrong, but isn't it used in `statistics_controller.rb` and `widgets_controller.rb`?

Reply:
The `before_action` calls this methods and so sets the variable. However, unless I am missing it, the actual variable isn't being used.

Reply:
The place where `@disable_navigation` was used was removed in this commit: https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3754/commits/ea07759a12b6518702e62914daae831a6fcdff95

I'm wondering if this was a mistake or if it has been replaced by `@enable_breadcrumb`, which would be the case if you either see navigation or breadcrumbs.

Reply:
I remember that too. I have removed it.